lib/rucio/common/schema/generic.py

```
@@ -197,7 +197,7 @@
     "description": "Client state",
     "type": "string",
     "enum": ['DONE', 'FAILED', 'PROCESSING', 'ALREADY_DONE', 'FILE_NOT_FOUND', 'FOUND_IN_PCACHE', 'DOWNLOAD_ATTEMPT',
-             'FAIL_VALIDATE', 'FOUND_ROOT']
+             'FAIL_VALIDATE', 'FOUND_ROOT', 'ServiceUnavailable']
```

Not really happy with CamelCase. But this is enough to suppress the errors mentioned in #5157. I was thinking to send this as a patch anyways and we address the issue with #5015 ? @bari12 thoughts?

---

On second thought, I'll apply the patch in the prod clusters now to suppress the errors. In case we update the code here, I will update the contents of the patch and rollout new k8s deployments to reflect the changes.